index.php

```diff
@@ -0,0 +1,24 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=damian-collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare("SELECT * FROM `albums`");
```

Comment:
It's generally best practice to list all the field names rather than using SELECT *